### ISSUE TITLE
Another module issue

### ISSUE THREAD

**Issue by flexoron** [trealla-prolog/trealla#587]
v2.55.41
```
?- [user]. p. q. end_of_file.
   true.
?- user:p.
   true.
?- user:q.
   true.
?- [user]. user:p. user:q. end_of_file.
Warning: overwriting 'p'/0
Warning: overwriting 'q'/0
   true.
?- user:p.
   false. % unexpected
?- user:q.
   false. % unexpected
?- listing.
user:p.
user:q.
   true.
?- 
```

**Comment by infradig** [comment]
$ tpl
?- [user]. p. q. end_of_file.
   true.
?- listing.
p.
q.
   true.
?- [user]. user:p. user:q. end_of_file.
Warning: overwriting 'p'/0
Warning: overwriting 'q'/0
   true.
?- listing.
user:p.
user:q.
   true.

**Comment by infradig** [comment]
Dunno why the module is still prepended.

**Comment by infradig** [comment]
More module mayhem:

$ tpl
?- assertz(user:p).
   true.
?- listing.
user:p.
   true.
?- p.
   false.
?- user:p.
   false.

**Comment by flexoron** [comment]
```
$ tpl 
?- [user]. p. q. end_of_file.
   true.
?- p.
   true.
?- user:p.
   true.
?- 'p'.
   true.
?- user:'p'.
   throw(error(existence_error(procedure,'3\x18\5'/0),'3\x18\5'/0)).
?- 
```

**Comment by flexoron** [comment]
This issue(#587) has been fixed with v2.55.42. Many thanks.
